prisma/schema.prisma

```diff
@@ -127,7 +127,7 @@ model Pokemon {
   specialDefense Int
   speed          Int
   ratings        RatingsOnPokemon[]
-  averageRating  Int
+  averageRating  Int?
```

Comment:
Be careful with this - it sounds like this should be a computed value based on all of the ratings a pokemon has. You'll have to maintain this value on the model, updating it every time a review gets added, removed or updated... which means you have some [tight coupling](https://www.educba.com/coupling-in-software-engineering/) in your application. In general, you don't want things to depend on each other.

This doesn't affect whether this PR is approved or not, just something to think about as you'll have to keep it in mind for almost everything you design/build.

Reply:
We will discuss further. We agree with you